README.md

```diff
@@ -42,7 +42,7 @@ yarn add web3
 ## Migration Guide
 
 -   [Migration Guide from Web3.js 1.x to 4.x](https://docs.web3js.org/docs/guides/web3_migration_guide)
-    Breaking changes are listed in migration guide and its first step for migrating from Web3.js 1.x to 4.x. If there is any question or discussion feel free to ask in discord, and in case of any bug or new feature request [open issue](https://github.com/web3/web3.js/issues/new) or create a pull request for [contributions](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).
+    Breaking changes are listed in migration guide and its first step for migrating from Web3.js 1.x to 4.x. If there is any question or discussion feel free to ask in [Discord](https://discord.gg/yjyvFRP), and in case of any bug or new feature request [open issue](https://github.com/web3/web3.js/issues/new) or [create a pull request]('https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request') for [contributions](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).
```

Comment:
```suggestion
    Breaking changes are listed in migration guide and its first step for migrating from Web3.js 1.x to 4.x. If there is any question or discussion feel free to ask in [Discord](https://discord.gg/yjyvFRP), and in case of any bug or new feature request [open issue](https://github.com/web3/web3.js/issues/new) or create a pull request for [contributions](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).
```

Reply:
I am new in GitHub. In files changed tab, I can see the changes as GitHub doc link is removed. so, you already changed it ? if yes, can owners can merge it?please guide me.

Reply:
@Upendra-Jaiswal no worries, what I did was make a `suggestion` to remove the code. In order for that changes to be applied, you'd need to click the button that says `Commit suggestion`, but there's no need to do that since you've already committed the changes in your [latest commit](https://github.com/web3/web3.js/pull/6073/commits/15164cb08299ea55840f97ba4b33d8da36b38ba4)